KUBECON.md

```diff
@@ -0,0 +1,79 @@
+[![](https://lh3.googleusercontent.com/AYP4fXhyrJsIpMpMOvUlfkie7YcP3AKpQauALZJBLgUMAaqIIba-xY4rWkL3ZngrY4XifKT7UyX_61uQR9QFsNI=w16383)Flux and Flagger @ KubeCon Paris 2024](/view/flux-kubecon-paris-2024/home)
+
+# KubeCon activities!
+
+![](https://lh6.googleusercontent.com/sjzEHiaFZnqzAv3-k96b6fi3IHZ013TykpUQBTkkHcVxXOr9Ng5X3XfzWKQultVinyxqkM6wORetAg8pN4Rd6Y7FZwyz7fCDc4w5TavrXBol9oYyDdPPPZUL69Q2E8JqXQ=w1280)
+
+# [Copy heading link](\#h.cn9lzphxuxjy)    KubeCon Paris 2024
```

Comment:
What are these copy header links?

Reply:
They are an artifact of the Google Sites page. I had manually removed them in the first iteration of the doc, that did not use html2md - they should not be in the final output so I think the script needs at least one more revision. (4ecbf0d)

They are easier to filter out now thanks to the differences in format between the previous tool and html2md


In the original Sites page, they are this handle (text shown when mouseover):
<img width="325" alt="Screenshot 2024-02-22 at 3 45 26 PM" src="https://github.com/fluxcd/community/assets/3286998/e1cf06e2-3947-4455-b56a-b84d302041a2">

In our website version, there's no need for that, since headers are automatically turned into anchors by markdown

The script is updated so that Copy header link will be filtered out from the output.

Now that you mention it I have also added a filter now to translate the top link from `/view/flux-kubecon-paris-2024/home` to `/kubecon`